lib/src/cupertino/cupertino_controls.dart

```diff
@@ -170,13 +170,18 @@ class _CupertinoControlsState extends State<CupertinoControls>
         decoration: BoxDecoration(
             color: const Color(0x96000000),
             borderRadius: BorderRadius.circular(10.0)),
-        child: Text(
-          currentSubtitle.first!.text,
-          style: const TextStyle(
-            fontSize: 18,
-          ),
-          textAlign: TextAlign.center,
-        ),
+        child: currentSubtitle.first!.text is InlineSpan
```

Comment:
`is` and `as` should be avoided. Instead provide directly the text as a Widget and you don't need this.